.github/workflows/test.yml

```diff
@@ -6,35 +6,32 @@ on:
 
 jobs:
   build:
-    name: Python ${{ matrix.python-version }} / ${{ matrix.tox-environment }}
+    name: Python ${{ matrix.python-version }}
     runs-on: ubuntu-latest
 
     strategy:
       fail-fast: false
       matrix:
         python-version:
+          - "3.7"
           - "3.8"
           - "3.9"
           - "3.10"
+          - "pypy-3.7"
           - "pypy-3.8"
-        tox-environment:
-          - django32-alchemy-mongoengine
-          - django40-alchemy-mongoengine
-        include:
-          - python-version: "3.7"
-            tox-environment: django22-alchemy-mongoengine
-          - python-version: "pypy-3.7"
-            tox-environment: django22-alchemy-mongoengine
-          - python-version: "3.7"
-            tox-environment: django32-alchemy-mongoengine
-          - python-version: "pypy-3.7"
-            tox-environment: django32-alchemy-mongoengine
```

Comment:
After all other issues have been resolved, we need to revert this change and deal with the problems testing Postgres with PyPy.

Reply:
Yeah. This was a major hurdle for me. tox+github actions don't play very since with one another :/

Reply:
I've remedied the issue for running only one python version per worker, we now do one worker per py+dj+db -- just as before.

I don't know what issue you're referring to. Can you elaborate? From my perspective, pypy + pg are working just fine. What am i missing?

Reply:
I thought that bullet point regrading pg and sqlite was related to the PyPy build problems. Nevermind.

Reply:
Very good. I will resolve this conversation